python/ray/projects/scripts.py

```diff
@@ -0,0 +1,101 @@
+import logging
+import os
+import sys
+from shutil import copyfile
```

Comment:
Is this available by default? Are we using it anywhere else?

Reply:
Yes. The following files use it:

```
python/setup.py
rllib/tests/test_checkpoint_restore.py
rllib/tests/test_io.py
python/ray/projects/scripts.py
python/ray/tests/test_basic.py
python/ray/tests/test_autoscaler.py
python/ray/tests/test_tempfile.py
python/ray/tune/trainable.py
python/ray/tune/tests/test_tune_restore.py
python/ray/tune/tests/test_trial_scheduler.py
python/ray/tune/tests/test_experiment_analysis.py
python/ray/tune/tests/test_trial_runner.py
python/ray/tune/tests/test_cluster.py
python/ray/tune/schedulers/pbt.py
```

I think we can safely assume it exists